src/main/java/duke/Task.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+/**
+ * Task class to represent a generic task.
+ *
+ * @author Sheryl Kong (A0240686Y)
+ */
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    private static int taskCount = 0;
```

Comment:
Interesting way of keeping track of the number of tasks in the list :)
Would it be more appropriate to have this in the TaskList class? Or perhaps use the `arraylist.size()` method?